src/script/bitcoinconsensus.h

```diff
@@ -33,15 +33,15 @@ extern "C" {
 
 #define BITCOINCONSENSUS_API_VER 1
 
-typedef enum bitcoinconsensus_error_t
+using bitcoinconsensus_error = enum bitcoinconsensus_error_t
```

Comment:
In commit "tidy: enable modernize-use-using" (313241d757c022d978d87e1abd9ab4bb68d45986)

Probably should revert this change if this header is supposed to be usable from `C` code. I assume `extern "C" {` above means it should be.